eZ/Bundle/EzPublishRestBundle/EventListener/CsrfListener.php

```diff
@@ -133,18 +133,36 @@ protected function isMethodSafe($method)
      * @param string $route
      *
      * @return bool
+     *
+     * @deprecated Deprecated since 6.5. Use isSessionRoute() instead.
      */
     protected function isLoginRequest($route)
     {
-        return $route === 'ezpublish_rest_createSession';
+        return $route === 'ezpublish_rest_createSession' ||
+               $route === 'ezpublish_rest_refreshSession';
+    }
+
+    /**
+     * Tests if a given $route is a session management one.
+     *
+     * @param string $route
+     *
+     * @return bool
+     */
+    protected function isSessionRoute($route)
+    {
+        return in_array(
+            $route,
+            ['ezpublish_rest_createSession', 'ezpublish_rest_refreshSession', 'ezpublish_rest_deleteSession']
+        );
     }
 
     /**
-     * @param GetResponseEvent $event
+     * @param Request $request
      *
      * @return bool
      */
-    protected function checkCsrfToken(Request $request)
+    protected function hasCsrfToken(Request $request)
```

Comment:
Unintended change, actually. Got confused with the ones from the controller, reverting.